.github/workflows/link-smoke-tests.yaml

```diff
@@ -0,0 +1,99 @@
+name: Lint and Smoke Tests
+
+on:
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  push:
+    branches:
+      - master
+
+env:
+  TIMEOUT: 15m
+  TYK_OSS_NS: tyk-oss
+  TYK_DATAPLANE_NS: tyk-dataplane
+  TYK_STACK_NS: tyk-stack
```

Comment:
`TYK_STACK_NS` is not used in smoke tests
```suggestion
```

Reply:
Removed